app/gui/suggestion-database/src/entry.rs

```diff
@@ -88,7 +88,7 @@ pub struct ModuleSpan {
 /// In order to make icon definitions more readable for non-programmer users, the builtin icon name
 /// is allowed to be formatted in arbitrary casing. Either `SNAKE_case`,`camelCase`, `Pascal_Case`
 /// etc. is allowed.
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize)]
```

Comment:
I assume this is `serde::Serialize` capability is required to convert these structures to JSON.

Reply:
Yep, that's right, we're automatically deriving serialization API for these structs.